src/form/form.tsx

```diff
@@ -133,6 +134,7 @@ class Form<MODEL extends object> extends React.Component<Props<MODEL>, State<MOD
         this.validationOptions = Object.assign(this.props.validationOptions || {}, getValidationOptionsDefaults());
 
         this.onSubmit = this.onSubmit.bind(this);
+        this.onUpdate = throttle(this.onUpdate.bind(this), 500);
```

Comment:
Please explain your use case and let me know what you are trying to achieve with this. This looks to me like a workaround.